.eslintrc.json

```diff
@@ -0,0 +1,26 @@
+{
+    "env": {
+      "browser": true,
+      "es6": true,
+      "jest": true
+    },
+    "parser": "babel-eslint",
+    "parserOptions": {
+      "ecmaVersion": 2018,
+      "sourceType": "module"
+    },
+    "extends": ["airbnb-base"],
+    "rules": {
+      "no-shadow": "off",
+      "eqeqeq": "off",
+      "no-param-reassign": "off",
+      "eol-last": "off",
+      "import/extensions": [ 1, {
+        "js": "always", "json": "always"
+      }]
+    },
```

Comment:
- [ ] Please use the exact eslint rules listed [here](https://github.com/microverseinc/linters-config/blob/master/javascript/.eslintrc.json)